torch/quantization/fx/quantization_patterns.py

```diff
@@ -775,14 +803,23 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
 @register_quant_pattern(torch.nn.InstanceNorm3d)
 @register_quant_pattern(torch.nn.LayerNorm)
 @register_quant_pattern(torch.nn.SiLU)
+# we currently only support reference patterns for these ops so they have been removed
+# until they receive a proper fp16 kernal. To use the reference pattern, use a custom qconfig
+# @register_quant_pattern(torch.nn.GELU)
+# @register_quant_pattern(torch.nn.Softmax)
 @register_quant_pattern(torch.nn.functional.hardswish)
 @register_quant_pattern(torch.nn.functional.instance_norm)
 @register_quant_pattern(torch.nn.functional.layer_norm)
 @register_quant_pattern(torch.nn.functional.leaky_relu)
 @register_quant_pattern(torch.nn.functional.silu)
+# we currently only support reference patterns for these ops so they have been removed
```

Comment:
These were not present before, so I am not sure I follow the comment. Do we need it?

Reply:
the comment was requested https://github.com/pytorch/pytorch/pull/56004/files#r614173173

I think the intention is to clarify why its unregistered here despite it being explicitly supported in our list of ops/dtypes.